ops/roles/ota/vars/main.yml

```diff
@@ -1,2 +0,0 @@
-ota_app_name: ota
-ota_repository: https://github.com/ambanum/OpenTermsArchive.git
```

Comment:
If this was a shared default between production and dev, why not keep it? 🙂

Reply:
I believed it was really weird to have only those 2 variables in another lonely file

More over, the ota repository and app name should be changeable
- app name to be able to launche several instances on the same machine
- repository to make it possible to use a fork

Again, having dispateched variables feels not right to me but I can rollback if needed

Reply:
I agree with Martin, I think the factorization he made it's a good improvement. 
It is easier to discover the variables and it is possible to override these variables for each instance if necessary.